content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+
+<Box display="flex" mb={3} flexWrap="wrap" sx={{gap: 3}}>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624842-f3abb168-1d3d-4cdb-a167-2c78059653d5.png"
+    role="presentation"
+    width="456"
+  />
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624843-75a82099-b95f-41f6-b9a9-35adc3e1537e.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+A two-column layout can be used to:
+
+- Align the control with adjacent elements
+- Conserve vertical space
+- Group related controls in a way that is easy to scan
+
+### Fill available width
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Text as="p" mt="0">
+    A segmented control's width can stretch each button evenly to fill the width of the control's parent. This option is
+    intended to make the segmented control easier to adapt into different layouts.
+  </Text>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624849-a8d785f2-0a0c-42dd-915c-65e26a09eb64.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+### Adapting to narrow areas
+
+There are cases where a segmented control needs to be placed in a spot that is too narrow to accommodate all of its options. In these cases, the control can be adapted to fit in the narrow area without sacrificing usability.
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="center"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <figcaption>
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Convert to an action menu
+    </Text>
+    <Text as="p" mt="0">
+      Swapping out the segmented control for an action menu is especially helpful when:
+      <Box as="ul" mt={1} pl="1rem">
+        <li>The icons are cannot be understood without visible text labels</li>
+        <li>The control is stil too wide when text labels are hidden</li>
+      </Box>
+    </Text>
+  </figcaption>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624846-a906563a-0878-44ce-89d1-46905b190d20.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="center"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  m={0}
+  mb={5}
+  sx={{gap: 3}}
+>
+  <figcaption>
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Only show icons
+    </Text>
+    <Text as="p" mt="0">
+      If your segmented control uses icons that can be understood without a visible text label, then you can hide the
+      text labels and rely only on icons. The text from the hidden label will still be visible in a tooltip.
+    </Text>
+  </figcaption>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624846-a906563a-0878-44ce-89d1-46905b190d20.png"
```

Comment:
This image seems to be repeated from the one [above](https://github.com/primer/design/pull/276/files#diff-28ec7427197ac7b3c80bca64eb9536b47e0dc24cf4e03d2e6a95c9c1b58b1f10R305).

Reply:
Oops. Will fix.